### PR TITLE
Fix drag over coloring

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

The coloring for dragover divs had gone astray to contain only red or green. Now when someone requests drag over regions, the places where a track can be added are colored blue, places where it can't are colored red and the region where the mouse hovers are colored green.

> Why is it necessary?

To distinguish allowed regions (e.g. top, left, bottom, right for a line track) vs. disallowed (e.g. 'center' for a line track).

> Screenshots

For a line track: correct

image

vs... all red.

Fixes #___

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [ ] Unit tests added or updated
- [x] Documentation added or updated
- [ ] Example(s) added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [ ] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by flekschas** [comment]
Why is "Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)" checked but the PR doesn't have any label? 🤔 Is this a bug fix PR or an enhancement?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+## v1.6.6
+
+- Fixed the drag over colors
```
Comment (flekschas) [inline CHANGELOG.md]: Can you elaborate a bit more? I don't know what this means.

File: app/scripts/DragListeningDiv.js [hunk 2]
```diff
@@ -18,7 +18,13 @@ class DragListeningDiv extends React.Component {
 
   render() {
     // color red if not enabled, green if a track is not top and red otherwise
```
Comment (flekschas) [inline app/scripts/DragListeningDiv.js]: Very minor: This comment seems outdated not that there is blue.

Comment (pkerpedjiev) [reply app/scripts/DragListeningDiv.js]: Good catch. Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (pkerpedjiev) [reply CHANGELOG.md]: Spruced that up a bit.

**Comment by pkerpedjiev** [comment]
> Why is "Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)" checked but the PR doesn't have any label? 🤔 Is this a bug fix PR or an enhancement?

Sorry about that, I thought that referred to the milestones. Added a label just now.

**Pull request merged**